app/bot/openai/remark.go

```diff
@@ -0,0 +1,123 @@
+package openai
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"regexp"
+	"sort"
+	"strings"
+	"time"
+
+	"github.com/go-pkgz/notify"
+	tbapi "github.com/go-telegram-bot-api/telegram-bot-api/v5"
+)
+
+// remarkComment is json structure for a comment from Remark42
+type remarkComment struct {
+	ParentID string `json:"pid"`
+	Text     string `json:"text"`
+	User     struct {
+		Name     string `json:"name"`
+		Admin    bool   `json:"admin"`
+		Verified bool   `json:"verified,omitempty"`
+	} `json:"user"`
+	Score     int       `json:"score"`
+	Deleted   bool      `json:"delete,omitempty" bson:"delete"`
+	Timestamp time.Time `json:"time" bson:"time"`
+}
+
+// Render returns a string representation of the comment
+func (c remarkComment) Render() string {
```

Comment:
is this one should be exposed? same about GetLink func

Reply:
This is used in `summary_test.go`. I thought this required to be public. Fixed